### ISSUE TITLE
Compatible with import file from "..."?

### ISSUE THREAD

**Issue by mattoni** [webpack-contrib/file-loader#96]
Is there a way to support default exports with the file loader plugin?

Currently, you have to do:

`import * as file from "file.jpg"`;

Ideally, I'd like to be able to do:

`import file from "file.jpg"`

I don't think this is configurable, but wondering if it's possible?

Thanks!

**Comment by SpaceK33z** [comment]
Do you happen to be using webpack v2?

**Comment by AlexanderMattoni** [comment]
@SpaceK33z Yes, I'm using webpack v2. I've been running into this issue with a few other plugins as well lol.

**Comment by SpaceK33z** [comment]
Okay, yeah the loaders are not using ES Modules yet. We are definitely picking this up after the v2 release (or somewhat before it).

**Comment by AlexanderMattoni** [comment]
Awesome, thank you for the info. I'm going to just use require("...") for now.

**Comment by es6Test** [comment]
Would love to use imports.

**Comment by SpaceK33z** [comment]
See PR #99.

**Comment by alexgorbatchev** [comment]
Crossposting this from #99.

You can fix this with a custom loader:

```js
export default function (content) {
  if (this.cacheable) this.cacheable();
  return `
    Object.defineProperty(exports, "__esModule", { value: true });
    ${content.replace(/^module\.exports =/, 'exports.default =')}
  `;
}
```

**Comment by evilebottnawi** [comment]
@alexgorbatchev Is the problem still there? New version use `export default` https://github.com/webpack-contrib/file-loader/blob/master/src/index.js#L78

**Comment by michael-ciniawsky** [comment]
Fixed in ` >= v1.0.0` (`@beta`)
